packages/gatsby-plugin-utils/package.json

```diff
@@ -3,6 +3,31 @@
   "version": "3.3.0-next.0",
   "description": "Gatsby utils that help creating plugins",
   "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*.js": "./dist/*.js",
+    "./dist/*": "./dist/*.js",
```

Comment:
Why do we need both mappings?

Reply:
To support both `gatsby-plugin-utils/dist/types` & `gatsby-plugin-utils/polyfill-remote-file`. Else you'll get `gatsby-plugin-utils/dist/dist/types`